easybuild/easyconfigs/p/PyTorch-bundle/PyTorch-bundle-1.12.1-foss-2022a-CUDA-11.7.0.eb

```
@@ -0,0 +1,59 @@
+easyblock = 'PythonBundle'
+
+name = 'PyTorch-bundle'
+version = '1.12.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://pytorch.org/'
+description = """PyTorch with compatible versions of official Torch extensions."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [
+    ('RE2', '2022-06-01'),  # for torchtext
+]
+
+dependencies = [
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('PyTorch', version, versionsuffix),
+    ('Pillow-SIMD', '9.2.0'),  # for torchvision
+    ('SentencePiece', '0.1.97'),  # for torchtext
+    ('tqdm', '4.64.0'),  # for torchtext
+    ('double-conversion', '3.2.0'),  # for torchtext
+]
+
+use_pip = True
+
+exts_list = [
+    ('torchvision', '0.14.1', {
+        'source_urls': ['https://github.com/pytorch/vision/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['ced67e1cf1f97e168cdf271851a4d0b6d382ab7936e7bcbb39aaa87239c324b6'],
+    }),
+    ('torchtext', '0.14.1', {
+        # Disable bundled libraries to use those from EB: RE2, SentencePiece
+        'preinstallopts': "sed -i '/third_party/d;/BuildExtension/d' setup.py &&",
+        'source_urls': ['https://github.com/pytorch/text/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['fd1ef3da7d9c20408c740f7dc7d02ad52a6048b46368355a1a7326d3bc4f2e63'],
+    }),
+    ('torchaudio', '0.13.1', {
+        'source_urls': ['https://github.com/pytorch/audio/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['3a5c108a9ea31d0e6533a2cd6cbaf133efaefe4bdcb4b689183a33b991c95d12'],
+    }),
+    ('pytorch-ignite', '0.4.11', {
+        'source_urls': ['https://github.com/pytorch/ignite/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['33e7485cea3cac08d7a49a3a01d3013e1156128867602bff0a8e32b2c5ebd4e9'],
+    }),
+    ('torchdata', '0.5.1', {
+        'source_urls': ['https://github.com/pytorch/data/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['69d80bd33ce8f08e7cfeeb71cefddfc29cede25a85881e33dbae47576b96ed29'],
+    }),
```

```suggestion
    }),
    ('torch-tb-profiler', '0.4.1', {
        'modulename': 'torch.profiler',
        'sources': ['torch_tb_profiler-%(version)s.tar.gz'],
        'checksums': ['f2c7fb27d420be443ffde50ada655c19f76a245d21e7772de753196fd0967685'],
    }),
```
if you want to include torch-tb-profiler